setup.scripts/mysql/mailwatch-mysql.inc

```diff
@@ -101,4 +110,7 @@ function setup-admin-user() {
         ask "Enter password for the admin: " MWAdminPwd "pw"
         logprint "Create MailWatch web gui admin"
         mysql -u "$SqlUser" -p"$SqlPwd" -h "$SqlHost" "$SqlDb" --execute="REPLACE INTO users SET username = '$MWAdmin', password = MD5('$MWAdminPwd'), fullname = 'Admin', type = 'A';"
```

Comment:
Could you use something better than MD5?
As SHA512-CRYPT

Reply:
Its the same as in the docs
https://docs.mailwatch.org/install/installing.html

Reply:
Its may be something to discuss in mailwatch upstream?
MD5 is old and weak.

Reply:
yep